src/modules/partners/partner_service.ts

```diff
@@ -42,6 +42,7 @@ export namespace Partner {
         next_page: itemsLength ? items[itemsLength - 1].created_at : null,
         per_page: itemsLength || 0,
```

Comment:
consider the last update from @ayocodingit we can use `perPage` instead `itemsLength || 0`